Library/Homebrew/extend/os/linux/cleaner.rb

```diff
@@ -2,6 +2,10 @@ class Cleaner
   private
 
   def executable_path?(path)
-    path.elf? || path.text_executable?
+    if OS.cygwin?
```

Comment:
Changes such like should should be in an `extend/os/cygwin` layer.

Reply:
Yes, I started down that route, but had issues. I'll see if I can work them out.